### DISCUSSION TITLE
支援 Windows 11 ARM 版本

### DISCUSSION THREAD

**Discussion by shuwn** [rime/home#1080] [Q&A]
請問目前有支援 Windows 11 ARM 版本嗎？
我在 Windows 11 ARM 版本可以順利安裝完成，但是輸出內容仍然是英數，沒辦法輸出中文

**Comment by lotem** [comment]
相同問題：
https://github.com/rime/home/issues/925
https://github.com/rime/home/issues/942

**Reply by FEI352** [reply]
大佬，今年有支持的计划吗？
